src/cql/r4/survey_resources/source/Common_LogicLibrary.cql

```diff
@@ -285,8 +285,11 @@ define function FormattedQuestionnaireResponses(responseItems List<FHIR.Question
         // PC-PTSD score item 102017-1
         when I.linkId.value = '102017-1' then I.answer[0].value.value
         when I.linkId.value = '/102017-1' then I.answer[0].value.value
+<<<<<<< HEAD
+=======
         // STOP score item
         when I.linkId.value = 'STOP-score' then I.answer[0].value.value
+>>>>>>> 83eac68f53639a94e5c72320d14369805ebe59f3
```

Comment:
This looks like the remnants of a git conflict; unclear which version is the correct one though...